runners/wikiConsole/templates/createFile.asciidoc

```diff
@@ -0,0 +1,30 @@
+== Create the file <%= fileName; %>
+
+==== Prerequisites
+First, you need to open a command prompt in your current workspace. For Windows, you can use PowerShell and on Linux, you can use the Terminal.
+
+The folder where you want to create the file has to be existing. If the folder doesn’t exist you have to create it.
+
+.Operating system
+. Windows + 
+Execute the command to create a folder if not already existing `New-Item -ItemType Directory -Force -Path <%= filePath; %>`.
+. Linux + 
+Execute the command `mkdir -p <%= filePath; %>`. The `-p` flag won't throw an error if the target folder already exists and allows the creation of parent directories.
+
+=== Create <%= fileName; %> in a Terminal
```

Comment:
I think we need to be more consistent on how terminal is written. I've seen it in lowercase and camelcase in our templates.

Reply:
I think because sometimes if you refer to the Linux Terminal is a name and in other cases you are just referring to terminals. But then its still wrong in this case.

Reply:
I will use command prompt instead